docs/DeveloperGuide.md

```diff
@@ -396,14 +402,20 @@ The following sequence diagram shows how delete operation works when `execute(de
 
 ![DeleteSequence](images/implementation/DeleteSequence.png)
 
-<div markdown="span" class="alert alert-info">:information_source: 
-**Note:** 
+<div markdown="block" class="alert alert-info">
+
+:information_source: **Note:** 
```

Comment:
I think can remove the colon for the note.

Reply:
ok removed!